tests/testthat/_snaps/attr-nodes/yarn.Rmd

```diff
@@ -0,0 +1,30 @@
+---
+title: "Untitled"
+author: "M. Salmon"
+date: "September 6, 2018"
+output: html_document
+---
+
+```{r setup, include=FALSE, eval=TRUE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# preface {#pre-face .unnumbered}
+
+hello
+
+I like {xml2} but of course {tinkr} is even cooler!
+
+Images that use pandoc style will have curlies with content that should be translated and should be protected.
+
+![a pretty kitten](https://placekitten.com/200/300){#kitteh alt='a picture of a kitten'}
+
+![a pretty puppy](https://placedog.net/200/300){#dog alt="a picture
+of a dog"}
+
+
+\[a span with attributes\]{.span-with-attributes
+style='color: red;'}
+
```

Comment:
why so many new lines

Reply:
I believe it's because of the `<softbreak>`: https://github.com/ropensci/tinkr/pull/76/files/3e133e5f6e6b35befef6d8fad20df6d87f8ad549?w=0#r973214227. I guess I was wrong in calling it benign -_-